optuna_dashboard/ts/action.ts

```diff
@@ -444,16 +444,44 @@ export const actionCreator = () => {
       })
   }
 
-  const tellTrial = (
+  const makeTrialFail = (studyId: number, trialId: number): void => {
+    const state = "Fail"
+    const message = `id=${trialId}, state=${state}`
```

Comment:
I think it doesn't need to declare the `state` variable.

```suggestion
    const message = `id=${trialId}, state=Fail`
```